tests/hypervisor/test_esx.py

```diff
@@ -262,6 +261,59 @@ def test_password(self, virtwho, function_hypervisor, esx_assertion):
                 and result['thread'] == 1
                 and assertion['null_multi_configs'] in result['error_msg'])
 
+    @pytest.mark.tier2
+    def test_encrypted_password(self, virtwho, function_hypervisor, esx_assertion,
+                                hypervisor_data, ssh_host):
+        """Test the encrypted_password= option in /etc/virt-who.d/test_esx.conf
+
+        :title: virt-who: esx: test encrypted_password option
+        :id: f07205a4-8f18-4c6b-8a2c-285664b59ed3
+        :caseimportance: High
+        :tags: tier2
+        :customerscenario: false
+        :upstream: no
+        :steps:
+            1. Delete the password option, encrypted password for the hypervisor.
+            2. Configure encrypted_password option with valid value, run the virt-who service.
+            3. Configure password option with invalid value.
+            4. Create another valid config file, run the virt-who service
+
+        :expectedresults:
+            2. Succeeded to run the virt-who, no error messages in the log info
+            3. Failed to run the virt-who service, find warning message:
+            'Option "encrypted_password" cannot be decrypted'
+            4. Find warning message: 'Option "encrypted_password" cannot be decrypted'
+        """
+        # encrypted_password option is valid value
```

Comment:
Suggest to separate the case to two ones because it's not a default option in /etc/virt-who.d/esx.conf, so it should be tested in the tier1.
Tier 1: `test_encrypted_password`
Tier 2: `test_invalid_encrypted_password`

Reply:
separate them to 2 classes